package.json

```
@@ -66,7 +66,7 @@
     "chai": "^4.2.0",
     "eslint": "^9.0.0",
     "eslint-config-prettier": "^10.0.0",
-    "ethers": "^6.13.4",
+    "ethers": "6.13.6-beta.1",
```

ethers [6.13.7](https://github.com/ethers-io/ethers.js/releases/tag/v6.13.7) was released, should this use that?
```suggestion
    "ethers": "^6.13.7",
```